src/main/java/com/metsci/laproc/uicomponents/GraphExporter.java

```diff
@@ -0,0 +1,87 @@
+package com.metsci.laproc.uicomponents;
+
+import com.metsci.glimpse.layout.GlimpseLayoutProvider;
+import com.metsci.glimpse.painter.decoration.LegendPainter;
+import com.metsci.glimpse.painter.info.CursorTextPainter;
+import com.metsci.glimpse.painter.plot.XYLinePainter;
+import com.metsci.glimpse.painter.shape.PolygonPainter;
+import com.metsci.glimpse.plot.SimplePlot2D;
+import com.metsci.glimpse.support.color.GlimpseColor;
+import com.metsci.laproc.plotting.*;
+import com.metsci.laproc.utils.IAction;
+
+/**
+ * Provides a layout
+ * Created by malinocr on 1/21/2017.
+ */
+public class GraphExporter implements GlimpseLayoutProvider {
```

Comment:
It seems like a lot of this code is very similar to code being used in GraphDisplayer. Can we condense this code at all?